extmod/nimble/modbluetooth_nimble.c

```diff
@@ -74,19 +74,19 @@ STATIC int ble_hs_err_to_errno(int err) {
 }
 
 // Note: modbluetooth UUIDs store their data in LE.
-STATIC ble_uuid_t *create_nimble_uuid(const mp_obj_bluetooth_uuid_t *uuid) {
+STATIC ble_uuid_t *create_nimble_uuid(const mp_obj_bluetooth_uuid_t *uuid, ble_uuid_any_t *storage) {
     if (uuid->type == MP_BLUETOOTH_UUID_TYPE_16) {
```

Comment:
Instead of this, you could factor `init_nimble_uuid` out of `create_nimble_uuid`. What do you think?

Reply:
Makes sense. I'll see what leads to the smallest code size and go with that.